plugin/pkg/global/resourcereferencemanager/admission.go

```diff
@@ -69,11 +73,13 @@ func Register(plugins *admission.Plugins) {
 type ReferenceManager struct {
 	*admission.Handler
 	gardenCoreClient           internalversion.Interface
+	gardenExternalCoreClient   externalcoreclientset.Interface
```

Comment:
Out of curiosity: why do we need both the internal and external core clients?
I think most plugins in k/k use external clients, but most of our plugins use internal ones.

Reply:
I saw the secrets part, and followed it. 
It works with internal client too.
Changed it.